composer.json

```diff
@@ -15,8 +15,8 @@
         "source": "https://github.com/woohoolabs/zen"
     },
     "require": {
-        "php": "^8.0.0",
-        "psr/container": "^1.0.0",
+        "php": ">=8.0",
```

Comment:
I wouldn't like to loosen the PHP compatibility requirement this much. If time comes (there are at least 2 more years), and this version of Zen is still compatible with PHP 9, then I'll release a patch version adding supporting for it.

Reply:
True, I had nightmares last night about rejecting the PR because of this change.